lib/matplotlib/table.py

```diff
@@ -410,7 +410,15 @@ def _do_cell_alignment(self):
 
     def auto_set_column_width(self, col):
 
-        self._autoColumns.append(col)
+        # check for col possibility on iteration
```

Comment:
This might be worded in another way. Suggestion:

> If `col` is iterable, concatenate to self._autoColumns. Otherwise, append it to self._autoColumns.